lib/app/app_settings/app_settings.dart

```diff
@@ -43,6 +43,12 @@ class AppSettingsData {
 
   final String mistralApiKey;
 
+  // for now check if the API key is not empty
+  bool isMistralApiKeyValid() {
+    final isNotBlank = mistralApiKey.trim().isNotEmpty;
+    return isNotBlank;
+  }
```

Comment:
What do you think about making a one-liner from it?
```suggestion
bool isMistralApiKeyValid() => mistralApiKey.trim().isNotEmpty;
```